certbot/plugins/common.py

```diff
@@ -262,15 +263,49 @@ def _setup_challenge_cert(self, achall, cert_key=None):
         return response
 
 
-# test utils used by certbot_apache/certbot_nginx (hence
+# utils used by certbot_apache/certbot_nginx (hence
 # "pragma: no cover") TODO: this might quickly lead to dead code (also
 # c.f. #383)
 
-def setup_ssl_options(config_dir, src, dest):  # pragma: no cover
-    """Move the ssl_options into position and return the path."""
-    option_path = os.path.join(config_dir, dest)
-    shutil.copyfile(src, option_path)
-    return option_path
+def install_ssl_options_conf(options_ssl, options_ssl_digest, mod_ssl_conf_src,
+    all_ssl_options_hashes, logger): # pragma: no cover
```

Comment:
Instead of passing in the logger, one should just be created for this file by putting:
```
logger = logging.getLogger(__name__)
```
at the top of the file. The only thing this affects is the string saying where the logging message was generated from in our file logs.